PerformanceCalculatorGUI/Components/ExtendedProfileScore.cs

```diff
@@ -111,15 +118,25 @@ private void load(RulesetStore rulesets)
                 {
                     positionChangeText.Text = $"{Score.PositionChange.Value:+0;-0;-}";
                 },
-                Children = new Drawable[]
+                Children = new[]
                 {
+                    ShowAvatar
+                        ? new ClickableAvatar(Score.SoloScore.User, true)
+                        {
+                            Masking = true,
+                            CornerRadius = ExtendedLabelledTextBox.CORNER_RADIUS,
+                            Size = new Vector2(avatar_size),
+                            Action = () => { host.OpenUrlExternally($"https://osu.ppy.sh/u/{Score.SoloScore.User?.Id}"); }
```

Comment:
```suggestion
                            Action = () => { host.OpenUrlExternally($"https://osu.ppy.sh/users/{Score.SoloScore.User?.Id}"); }
```

`/u/{id}` is a legacy URL, whilst there is a redirect from `/u` to `/users` I'd rather just avoid it altogether